hypervisor/src/mshv/mod.rs

```diff
@@ -370,11 +370,21 @@ impl hypervisor::Hypervisor for MshvHypervisor {
         self.create_vm_with_type(vm_type)
     }
     #[cfg(target_arch = "x86_64")]
+
     ///
     /// Get the supported CpuID
     ///
     fn get_supported_cpuid(&self) -> hypervisor::Result<Vec<CpuIdEntry>> {
-        Ok(Vec::new())
+        let mut cpuid: Vec<CpuIdEntry> = Vec::new();
+        let functions: [u32; 2] = [0x1, 0xb];
+
+        for fnc in functions {
+            cpuid.push(CpuIdEntry {
+                function: fnc,
+                ..Default::default()
+            });
+        }
```

Comment:
nit:

```
for function in functions {
    cpuid.push(CpuIdEntry {
          function,
          Default::default()
    })
}
```

Reply:
|
381 |             cpuid.push(CpuIdEntry {
    |                        ---------- while parsing this struct
382 |                   function,
383 |                   Default::default()
    |                   -------^^ expected one of `,`, `:`, or `}`
    |                   |
    |                   while parsing this struct field

Reply:
I think what Jinank means is you can use `function` instead of `fnc` in the loop.